runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -2318,7 +2329,7 @@ If you need to add more rule tables to this decision table spreadsheet, skip a r
 
 NOTE: Only the first worksheet in a spreadsheet workbook is processed as a decision table when you include the spreadsheet in your {PRODUCT} project. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
 
-After you include the decision table in your {PRODUCT} project, the rules are rendered as DRL rules like the following example, from the sample spreadsheet:
+After you include the decision table in your {PRODUCT} project, the rules are interpreted as DRL rules like the following example, from the sample spreadsheet:
```

Comment:
not using word 'rendered', as BAMOE does not have graphical editor for spreadsheet/DRL decisions